docs/security.rst

```diff
@@ -0,0 +1,12 @@
+.. _schemaregistry_security:
+
+Security Overview
+-----------------
+The Schema Registry currently only supports communication with a secure Kafka cluster over SSL. At this time, REST API and ZooKeeper security is not yet supported. Kafka SASL authentication is not supported yet either.
```

Comment:
REST API is a bit confusing here. May be we should say the only support request over http not https